transformations/longer_location_ner/transformation.py

```diff
@@ -0,0 +1,93 @@
+from typing import List, Tuple
+import random
+from interfaces.TaggingOperation import TaggingOperation
+from tasks.TaskTypes import TaskType
+
+"""
+A tagging implementation of NER systems.
+I am travelling to London. --> I am travelling to South/Central London.
+"""
+
+
+def pick_random_word(seed):
+    # pick random "pre" or "post" perturb type with respective word
+    random.seed(seed)
+    pre_word_list = ["East", "West", "South", "North", "Eastern", "Western", "Central"]
```

Comment:
Berkley 
--> Southern Berkley 
--> Northern Berkley
--> Northeast Berkley
--> Southwest Berkley etc.
--> South West Berkley (3 tags could be added here)
--> SE Berkley 

Secondary inter-cardinal directions (check here: https://en.wikipedia.org/wiki/Cardinal_direction)
(Northern North East or NNE I think and those like that)

Reply:
completed all suggested changes.